addons/marketing_card/security/ir.model.access.csv

```diff
@@ -0,0 +1,18 @@
+id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
+access_card_campaign_tag_user,card.campaign.tag.user,model_card_campaign_tag,marketing_card.marketing_card_group_user,1,1,0,0
+access_card_campaign_tag_manager,card.campaign.tag.manager,model_card_campaign_tag,marketing_card.marketing_card_group_manager,1,1,1,1
+
+access_card_campaign_user,card.campaign.user,model_card_campaign,marketing_card.marketing_card_group_user,1,1,1,1
+access_card_campaign_manager,card.campaign.manager,model_card_campaign,marketing_card.marketing_card_group_manager,1,1,1,1
+
+access_card_template_user,card.template.user,model_card_template,marketing_card.marketing_card_group_user,1,0,0,0
+access_card_template_manager,card.template.manager,model_card_template,marketing_card.marketing_card_group_manager,1,1,1,1
+
+access_card_campaign_element_user,card.campaign.element.user,model_card_campaign_element,marketing_card.marketing_card_group_user,1,1,1,1
+access_card_campaign_element_manager,card.campaign.element.manager,model_card_campaign_element,marketing_card.marketing_card_group_manager,1,1,1,1
+
+access_card_card_share_user,card.card.share.user,model_card_card_share,marketing_card.marketing_card_group_user,1,1,1,0
+access_card_card_share_manager,card.card.share.manager,model_card_card_share,marketing_card.marketing_card_group_manager,1,1,1,0
+
+access_card_card_user,card.card.user,model_card_card,marketing_card.marketing_card_group_user,1,1,1,0
```

Comment:
As far as I can tell, this model is created through controller and sudo'ed records, does the marketing card user need to have that much access?

(It's even questionable for the card.manager, but I guess we can leave it for debug purposes).